qutebrowser/browser/webkit/webpage.py

```diff
@@ -298,6 +302,27 @@ def on_load_started(self):
         else:
             self.error_occurred = False
 
+    @pyqtSlot()
+    def handle_clear_focus(self):
+        # Handle clearing focus when page is loaded
+        if not config.val.input.blur_on_load.enabled:
+            return
+        code = """
+            window.addEventListener("DOMContentLoaded",
+                () => document.activeElement.blur());
+            """
+        self.mainFrame().evaluateJavaScript(code)
+
+        if config.val.input.blur_on_load.delay < 0:
+            return
+
+        code = """
+            window.addEventListener("load",
+                () => setTimeout(
+                    () => document.activeElement.blur(), {}));
```

Comment:
I would be interested in a way to de-duplicate this code with the webengine implementation, but I'm not sure if that's possible, and it's probably not worth the effort if it's questionable.